test/webpage.test.ts

```diff
@@ -0,0 +1,109 @@
+import * as chai from "chai";
+import { expect } from "chai";
+import * as chaiAsPromised from "chai-as-promised";
+chai.use(chaiAsPromised);
+import * as sinon from "sinon";
+
+import * as ServerMock from "mock-http-server";
+
+import { join } from "path";
+
+import { promisify } from "util";
+import { readFile } from "fs";
+const readFilePromise = promisify(readFile);
+
+import { Webpage } from "../src/Webpage";
```

Comment:
Please, reorder the imports and move the other statements after them.

You should follow this order of imports, also by grouping similar ones together:
1. Absolute first
2. Relative then

Reply:
Reordered the imports and moved other statements underneath